sdk/textanalytics/Azure.AI.TextAnalytics/src/DetectLanguageInput.cs

```diff
@@ -44,10 +35,5 @@ public DetectLanguageInput(string id, string text)
         /// To remove this behavior, set to <see cref="None"/>.
         /// </summary>
         public string CountryHint { get; set; }
```

Comment:
Not part of this change, but why `CountryHint` is a get/set property and `Id` is a read-only property?

Reply:
We want people to only be able to set `Id` and `Text` once when creating the `DetectLanguageInput` object. These are the properties that make an object unique.
`CountryHint` is not mandatory and could be added or modified whenever